cmd/minikube/cmd/docker-env.go

```diff
@@ -427,7 +427,9 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 			// TODO: refactor to work with docker, temp fix to resolve regression
 			if cr == constants.Containerd {
 				// eventually, run something similar to ssh --append-known
-				appendKnownHelper(nodeName, true)
+				if err := appendKnownHelper(nodeName, appendKnown); err != nil {
```

Comment:
```suggestion
				if err := appendKnownHelper(nodeName, true); err != nil {
```
`appendKnown` is a flag from another command, it's impossible to set this to true.